app/controllers/api/v1/appointments_controller.rb

```diff
@@ -1,4 +1,26 @@
 class Api::V1::AppointmentsController < Api::Current::AppointmentsController
   include Api::V1::ApiControllerOverrides
   include Api::V1::SyncControllerOverrides
+
```

Comment:
We should also log to sentry when `other` cancel reason tries to overwrite a cancel reason. The chance of it happening is low, and we should capture it if it ever happens.

Reply:
I'm explicitly catching cases where updates happen to cancelled appointments and returning errors. And we log to sentry when we return errors, ya?